Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/apiserver/resthandler.go

```diff
@@ -69,8 +70,7 @@ func GetResource(r RESTGetter, ctxFn ContextFunc, namer ScopeNamer, codec runtim
 			return
 		}
 		if err := setSelfLink(result, req, namer); err != nil {
-			errorJSON(err, codec, w)
-			return
+			glog.Errorf("failure while setting SelfLink: %v", err)
```

Comment:
> ... inside setSelfLink

"And I must have blue window treatments with purple polkadots on my condemned house...."